main.py

```diff
@@ -92,12 +96,8 @@ def __init__(self, *args, **kwargs):
             sanitized_message = sanitize(message)
             log.debug(f"Sanitized {sanitized_message}, Original: {message}")
             try:
-                await rat_command.trigger(message=sanitized_message,
-                                          sender=user,
-                                          channel=channel)
-            except CommandNotFoundException:
-                # command was not found
-                log.debug("Command invoked from \"{message}\" not found.")
+                ctx = await Context.from_message(self, channel, user, message)
```

Comment:
Didn't you already make the context instance at line 87?

Reply:
Yeah, but it belongs here. fixed